### PR TITLE
Create display expiration date

### PR DESCRIPTION
**Pull request by EagleEye25**

### CODE REVIEW

File: src/app/core/market/api/template/template.model.ts [hunk 1]
```diff
@@ -175,6 +175,11 @@ export class Template {
     return this.object.FlaggedItem;
   }
 
+  get expiredAt(): any {
+    const expire = new DateFormatter(new Date(this.object.ListingItems[0].expiredAt)).dateFormatter(false).substr(0, 16);
```
Comment (zaSmilingIdiot) [inline src/app/core/market/api/template/template.model.ts]: If this.object does not have ListingItems or this.object.ListingItems is an empty array, then this line will throw an exception.

**Review by zaSmilingIdiot — CHANGES_REQUESTED** [review]
Requires a couple of checks that ensure existence before usage.

File: src/app/core/market/api/template/template.model.ts (hunk 1 above)
Comment (EagleEye25) [reply src/app/core/market/api/template/template.model.ts]: Completed

**Comment by vikas-cis-zz** [comment]
@EagleEye25 didn't saw your last update :P it is much better :D :D nevermind 

Thanks

**Review by zaSmilingIdiot — APPROVED** [review]

**Pull request merged**